prow/config.yaml

```diff
@@ -1,5 +1,26 @@
 plank:
-  job_url_template: 'https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/{{if eq .Spec.Type "presubmit"}}pr-logs/pull{{else if eq .Spec.Type "batch"}}pr-logs/pull{{else}}logs{{end}}{{if ne .Spec.Refs.Org ""}}{{if ne .Spec.Refs.Org "kubernetes"}}/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}{{else if ne .Spec.Refs.Repo "kubernetes"}}/{{.Spec.Refs.Repo}}{{end}}{{end}}{{if eq .Spec.Type "presubmit"}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else if eq .Spec.Type "batch"}}/batch{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
+  job_url_template: >-
```

Comment:
s/`>-`/`|-`/ use block style seems fine and can still pass the tests?

Reply:
Yeah, if the user of this field can handle newlines.

Reply:
No, newlines are not acceptable

Reply:
@stevekuznetsov why?

Reply:
Can't have newlines in the URL and the `{{-` syntax in Go templates won't snip them

Reply:
I guess we can post-process this field after we read in config to remove newlines

Reply:
hummmm - we probably want to have a unit test against the `job_url_template` in config.yaml (thought we had one, but)

Reply:
There is a unit test, that's not the problem. The problem is that the config_sort test is trying to sort something that isn't a job.

Reply:
load and dump is not smart enough :-( it does only *sort* the jobs https://github.com/kubernetes/test-infra/blob/master/jobs/config_sort.py#L90-L92